### PR TITLE
Rimworld.

### PR DESCRIPTION
**Pull request by Zuhayr**
- Added a new two-level map, rimworld, a collaborative project with Nienhaus that will serve as a testing map/distraction from water.
- Adds several unticked files used to compile/build the rimworld map rather than Europa.
- Adds numerous new natural turfs with associated flora.
- Allows trees to be cut down for logs.
- Allows decals to be applied to all simulated turfs.

### CODE REVIEW

File: code/modules/europa/turfs/natural.dm [hunk 1]
```diff
@@ -0,0 +1,73 @@
+var/list/grass_types = list(
+	/obj/structure/flora/ausbushes/sparsegrass,
+	/obj/structure/flora/ausbushes/fullgrass
+	)
+
+var/list/tree_types = list(
+	/obj/structure/flora/tree,
```
Comment (Zuhayr) [inline code/modules/europa/turfs/natural.dm]: This is invisible, needs to be removed.

**Comment by Nienhaus** [comment]
:+1:

**Comment by Zuhayr** [comment]
http://imgur.com/a/wT3wd Current state of the map at runtime.

**Comment by Nienhaus** [comment]
10/10 I'm loving it.

**Pull request merged**